From 6f2210178859f884e76d8f6655c2ed21329b371f Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 29 May 2024 21:55:00 +0200
Subject: [PATCH] Update setup.py

---
 setup.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/setup.py b/setup.py
index a51a842eaa..3c56ace4b1 100644
--- a/setup.py
+++ b/setup.py
@@ -136,7 +136,6 @@ def load_requirements(file_list: Optional[Union[str, List[str]]] = None) -> List
         "biomed": load_requirements("requirements/ingest/biomed.in"),
         "box": load_requirements("requirements/ingest/box.in"),
         "chroma": load_requirements("requirements/ingest/chroma.in"),
-        "clarifai": load_requirements("requirements/ingest/clarifai.in"),
         "confluence": load_requirements("requirements/ingest/confluence.in"),
         "delta-table": load_requirements("requirements/ingest/delta-table.in"),
         "discord": load_requirements("requirements/ingest/discord.in"),
@@ -169,9 +168,6 @@ def load_requirements(file_list: Optional[Union[str, List[str]]] = None) -> List
         "paddleocr": load_requirements("requirements/extra-paddleocr.in"),
         "embed-huggingface": load_requirements("requirements/ingest/embed-huggingface.in"),
         "embed-mixedbreadai": load_requirements("requirements/ingest/embed-mixedbreadai.in"),
-        "embed-octoai": load_requirements("requirements/ingest/embed-octoai.in"),
-        "embed-vertexai": load_requirements("requirements/ingest/embed-vertexai.in"),
-        "embed-voyageai": load_requirements("requirements/ingest/embed-voyageai.in"),
         "openai": load_requirements("requirements/ingest/embed-openai.in"),
         "bedrock": load_requirements("requirements/ingest/embed-aws-bedrock.in"),
         "databricks-volumes": load_requirements("requirements/ingest/databricks-volumes.in"),
